api/app/controllers/magic_link.py

```diff
@@ -45,7 +45,7 @@ def send_magic_link(self, email_address: str):
         _ = MagicLink.clear_for_user(self.db_session, user.uid)
         logger.debug("creating magic link for user %s", user.id)
         _ = MagicLink(
-            user_uid=user.uid,
+            fkey_user_uid=user.uid,
```

Comment:
all the underlying `uid` foreign keys are now prefixed `fkey_` and renamed to `<object>_uid` so there's no confusion with `id` which is the string property.